### PR TITLE
Add trilo xtest to users [SKIP_CI]

### PR DESCRIPTION
**Pull request by dreamer-coding-555**
Adding an entry to user list

### CODE REVIEW

File: docs/markdown/Users.md [hunk 1]
```diff
@@ -160,6 +160,7 @@ format files
  - [Terminology](https://github.com/billiob/terminology), a terminal emulator based on the Enlightenment Foundation Libraries
  - [Tilix](https://github.com/gnunn1/tilix), a tiling terminal emulator for Linux using GTK+ 3
  - [Tizonia](https://github.com/tizonia/tizonia-openmax-il), a command-line cloud music player for Linux with support for Spotify, Google Play Music, YouTube, SoundCloud, TuneIn, Plex servers and Chromecast devices
+ - [Trilobite XUnit Test](https://github.com/trilobite-stdlib/trilo-xtest-c), That test framework you wish you had when you started learning C. Has unit testing, UI testing, UX testing, AI testing, benchmarks, mocking, command-line interface, and much more.
```
Comment (Volker-Weissmann) [inline docs/markdown/Users.md:163]:
```suggestion
 - [Trilobite XUnit Test](https://github.com/trilobite-stdlib/trilo-xtest-c), a framework for testing C code
```

**Comment by dreamer-coding-555** [comment]
Alright changed the description

**Comment by Volker-Weissmann** [comment]
LGTM

**Comment by eli-schwartz** [comment]
(In general, changes like this should be squashed.)

**Pull request merged**